tests/transient_storage.rs

```diff
@@ -0,0 +1,176 @@
+// //! storage tests
```

Comment:
if you want to test WASM / opcode related stuffs directly,  you need at least fill the logic of  `compiler/filetests/wat/transient_storage/basic.wat` then work based on it

Reply:
yeahh but i don't really understand the wat